cli_tools/common/utils/storage/resource_location_retriever.go

```diff
@@ -28,44 +28,45 @@ var (
 	// for each multi region, regions are sorted by cost in ascending order as of the time of writing
 	// this code.
 	multiRegions = map[string][]string{
```

Comment:
- Which cost is sorted by?
- Can you put a link to the page where someone can find that cost.
- For future proofing, can we switch to an API?
   - For example, on https://cloud.google.com/storage/docs/locations#location-dr , I see `NORTHAMERICA-NORTHEAST1`

Reply:
Which API are you referring to? I couldn't find an API for region to multi-region mapping.

Reply:
Added links to docs and expanded on the description.

I don't believe there's an API for region to multiregion mapping. The best I found is the docs you linked to. 

Regarding NORTHAMERICA-NORTHEAST1, it's in Canada, so I don't believe it should belong to "US" multi-region, although I couldn't find any docs that it doesn't belong there.